### ISSUE TITLE
Find projects that are beginners friendly and add to this awesome list for Hacktoberfest 2020

### ISSUE THREAD

**Issue by OtacilioN** [OtacilioN/awesome-hacktoberfest-2020#179]
Welcome to Hacktoberfest 2020! 🎉

**Comment by fant0me** [comment]
opportunity for  developrs

**Comment by jabhishek87** [comment]
can you please assign it to me ?

**Comment by OtacilioN** [comment]
Yes you can @jabhishek87, but remember that only pull requests opened by 1 October counts.

**Comment by lucasace** [comment]
@OtacilioN Is it possible for you to add this as well https://github.com/lucasace/Best11-Fantasycricket

**Comment by Mahikta** [comment]
Can you please assign it to me?

**Comment by sakshi006** [comment]
Hey!
Can i work on this issue?

**Comment by emmanuelJet** [comment]
Can I add two beginners friendly repository participating in Hacktoberfest?

**Comment by Dhananjay1999singh** [comment]
Please assign this to me

**Comment by vanestorz** [comment]
could you please explain what kind of repo declared as "beginner friendly"? or it could be any repo as long as it maintained properly?
